django_ilmoitin/dummy_content.py

```diff
@@ -0,0 +1,5 @@
+class DummyContent:
```

Comment:
would maybe call it `DummyContext`, since `content` is what the admin will manage in the template